hack/asciidoc-flows-gen.sh

```
@@ -15,23 +15,23 @@ echo -e '[cols="1,1,3,1,1,1,1",options="header"]' >> $ADOC
+nbfields=$(./bin/yq '.fields | length' $FE_SOURCE)
 lokiLabels=$(cat $LOKI_LABEL_SOURCE)
 cardinalityMap=$(cat $CARDINALITY_SOURCE)
 otelMap=$(cat $OTEL_SOURCE)
 errors=""
 
 for i in $(seq 0 $(( $nbfields-1 )) ); do
-  frontEntry=$(yq ".fields | sort_by(.name) | .[$i]" $FE_SOURCE)
-  name=$(printf "$frontEntry" | yq ".name")
-  type=$(printf "$frontEntry" | yq ".docType")
+  frontEntry=$(./bin/yq ".fields | sort_by(.name) | .[$i]" $FE_SOURCE)
+  name=$(printf "$frontEntry" | ./bin/yq ".name")
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)